<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_goToHome">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>홈으로</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_startExercise">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>운동 시작</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_myData">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>분석하기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_board">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>게시판</string>
    </property>
   </widget>
   <widget class="QLabel" name="tablet_frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-70</x>
      <y>-30</y>
      <width>1301</width>
      <height>871</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="main_frame.qrc">:/main_frame/img/tablet_frame.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="sign_img">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>1091</width>
      <height>671</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="main_frame.qrc">:/exercise/img/ex_form1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>301</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>370</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{ background-color: rgba(255, 255, 255, 0); }
QPushButton:hover{}
QPushButton:l:hover:pressed {}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_seleted_ex_0">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>310</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tmon몬소리 Black</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>선택 운동 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_seleted_ex_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>400</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tmon몬소리 Black</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>모범 영상</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_seleted_ex_1">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>310</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>야놀자 야체 B</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>덤벨 데드 리프트</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_seleted_ex_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>350</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>야놀자 야체 B</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>이 운동은 허벅지 뒤와 엉덩이, 그리고 허리에 효과적인 운동입니다.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_seleted_ex_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>580</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>야놀자 야체 B</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>진행 사항</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_logout">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>68</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{ background-color: rgba(255, 255, 255, 0); }
QPushButton:hover{}
QPushButton:l:hover:pressed {}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_help">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{ background-color: rgba(255, 255, 255, 0); }
QPushButton:hover{}
QPushButton:l:hover:pressed {}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_user_photo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="main_frame.qrc">:/main/img/user.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_train">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>240</width>
      <height>300</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="main_frame.qrc">:/img_logo/img/logo.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lb_Dday_num">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tmon몬소리 Black</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_Dday">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tmon몬소리 Black</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>D-day</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_selectDay">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>450</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>D-day 변경</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_profile_change">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>사진 변경</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_model">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>400</y>
      <width>311</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="main_frame.qrc">:/exercise/img/ex_ham_model.JPG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_model_start">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>440</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>▶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_model_stop">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>440</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>■</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_cnt_box">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>610</y>
      <width>450</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="main_frame.qrc">:/exercise/img/img_zero.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_train_start">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>640</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>▶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_train_save">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>640</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tmon몬소리 Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_train_stop">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>640</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>■</string>
    </property>
   </widget>
   <zorder>tablet_frame</zorder>
   <zorder>sign_img</zorder>
   <zorder>btn_goToHome</zorder>
   <zorder>btn_startExercise</zorder>
   <zorder>btn_myData</zorder>
   <zorder>btn_board</zorder>
   <zorder>calendarWidget</zorder>
   <zorder>btn_2</zorder>
   <zorder>lb_seleted_ex_0</zorder>
   <zorder>lb_seleted_ex_3</zorder>
   <zorder>lb_seleted_ex_1</zorder>
   <zorder>lb_seleted_ex_2</zorder>
   <zorder>lb_seleted_ex_4</zorder>
   <zorder>btn_logout</zorder>
   <zorder>btn_help</zorder>
   <zorder>lb_user_photo</zorder>
   <zorder>lb_train</zorder>
   <zorder>lb_Dday_num</zorder>
   <zorder>lb_Dday</zorder>
   <zorder>btn_selectDay</zorder>
   <zorder>btn_profile_change</zorder>
   <zorder>lb_model</zorder>
   <zorder>btn_model_start</zorder>
   <zorder>btn_model_stop</zorder>
   <zorder>lb_cnt_box</zorder>
   <zorder>btn_train_start</zorder>
   <zorder>btn_train_save</zorder>
   <zorder>btn_train_stop</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="main_frame.qrc"/>
 </resources>
 <connections/>
</ui>
